src/heretic/doomdef.h

```diff
@@ -527,7 +527,11 @@ extern boolean DebugSound;      // debug flag for displaying sound info
 extern int GetWeaponAmmo[NUMWEAPONS];
 
 extern boolean demorecording;
+extern boolean lowres_turn; // Truncate angleturn in ticcmds to nearest 256.
+                            // Used when recording Vanilla demos in netgames.
```

Comment:
Please don't write multi-line end-of-line comments; end-of-line comments like this are okay in this context but if it requires more than one line of explanation then place the comment before the statement.